remo/profiles/fixtures/demo_users.json

```diff
@@ -3,7 +3,7 @@
         "pk": 4,
         "model": "auth.user",
         "fields": {
-            "username": "admin",
+            "username": "nikos-admin",
```

Comment:
Could you keep this as "admin"?

Reply:
`admin` was the conflicting key.

Reply:
I know, but it was conflicting with the one further down, right? If you change this one here to "nikos-admin", you won't have any with "admin" anymore. Correct?

Reply:
Yeah, that is indeed correct. But the fixtures weren't still loading and showing Integrity error with `username`:

```
./manage.py loaddata demo_users
No handlers could be found for logger "product_details"
Traceback (most recent call last):
  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/sanyamkhurana/personal/remo/venv/lib/python2.7/site-packages/django/core/management/__init__.py", line 354, in execute_from_command_line
    utility.execute()
  File "/Users/sanyamkhurana/personal/remo/venv/lib/python2.7/site-packages/django/core/management/__init__.py", line 346, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/sanyamkhurana/personal/remo/venv/lib/python2.7/site-packages/django/core/management/base.py", line 394, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/sanyamkhurana/personal/remo/venv/lib/python2.7/site-packages/raven/contrib/django/management/__init__.py", line 41, in new_execute
    return original_func(self, *args, **kwargs)
  File "/Users/sanyamkhurana/personal/remo/venv/lib/python2.7/site-packages/django/core/management/base.py", line 445, in execute
    output = self.handle(*args, **options)
  File "/Users/sanyamkhurana/personal/remo/venv/lib/python2.7/site-packages/django/core/management/commands/loaddata.py", line 60, in handle
    self.loaddata(fixture_labels)
  File "/Users/sanyamkhurana/personal/remo/venv/lib/python2.7/site-packages/django/core/management/commands/loaddata.py", line 100, in loaddata
    self.load_label(fixture_label)
  File "/Users/sanyamkhurana/personal/remo/venv/lib/python2.7/site-packages/django/core/management/commands/loaddata.py", line 157, in load_label
    obj.save(using=self.using)
  File "/Users/sanyamkhurana/personal/remo/venv/lib/python2.7/site-packages/django/core/serializers/base.py", line 173, in save
    models.Model.save_base(self.object, using=using, raw=True)
  File "/Users/sanyamkhurana/personal/remo/venv/lib/python2.7/site-packages/django/db/models/base.py", line 762, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "/Users/sanyamkhurana/personal/remo/venv/lib/python2.7/site-packages/django/db/models/base.py", line 827, in _save_table
    forced_update)
  File "/Users/sanyamkhurana/personal/remo/venv/lib/python2.7/site-packages/django/db/models/base.py", line 877, in _do_update
    return filtered._update(values) > 0
  File "/Users/sanyamkhurana/personal/remo/venv/lib/python2.7/site-packages/django/db/models/query.py", line 580, in _update
    return query.get_compiler(self.db).execute_sql(CURSOR)
  File "/Users/sanyamkhurana/personal/remo/venv/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 1062, in execute_sql
    cursor = super(SQLUpdateCompiler, self).execute_sql(result_type)
  File "/Users/sanyamkhurana/personal/remo/venv/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 840, in execute_sql
    cursor.execute(sql, params)
  File "/Users/sanyamkhurana/personal/remo/venv/lib/python2.7/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/Users/sanyamkhurana/personal/remo/venv/lib/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/Users/sanyamkhurana/personal/remo/venv/lib/python2.7/site-packages/django/db/utils.py", line 98, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/Users/sanyamkhurana/personal/remo/venv/lib/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/Users/sanyamkhurana/personal/remo/venv/lib/python2.7/site-packages/django/db/backends/mysql/base.py", line 124, in execute
    return self.cursor.execute(query, args)
  File "/Users/sanyamkhurana/personal/remo/venv/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/Users/sanyamkhurana/personal/remo/venv/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
django.db.utils.IntegrityError: Problem installing fixture '/Users/sanyamkhurana/personal/remo/remo/profiles/fixtures/demo_users.json': Could not load auth.User(pk=4): (1062, "Duplicate entry 'admin' for key 'username'")
```

Reply:
I'm not sure why the `pk` is same in both of the entries. Let me see if we can actually clear this data, or may be have a better way to generate dummy data.

Reply:
I think the first step is to clean the existing fixture up so it works again. Then we can work on making it more useful. I think the conflicting admin one was because at the first run it already created it until it failed. Then you already had an "admin", and then when you run the script again you get the error you mentioned. Can you delete all data and try again?

Reply:
Oh yes, you're correct! 

Just fixed that.